secretstores/hashicorp/vault/vault.go

```diff
@@ -49,15 +52,37 @@ const (
 	defaultVaultKVPrefix         string = "dapr"
 	vaultHTTPHeader              string = "X-Vault-Token"
 	vaultHTTPRequestHeader       string = "X-Vault-Request"
+	vaultEnginePath              string = "enginePath"
+	vaultValueType               string = "vaultValueType"
+	versionID                    string = "version_id"
```

Comment:
This is not following the same naming convention. Please, update.

Reply:
There is no breaking change, the default value keeps it backward compatible.

`enginePath` and `vaultValueType` are used in component metadata, these fields are in Camel-Case.

`version_id` is used in query parameters, and other secret stores which support the version are using this key. the doc link: https://docs.dapr.io/reference/api/secrets_api/#query-parameters